backend/views/syson-diagram-general-view/src/main/java/org/eclipse/syson/diagram/general/view/nodes/CompartmentNodeDescriptionProvider.java

```diff
@@ -173,7 +174,7 @@ private NodeTool createCompartmentNodeTool(EClassifier type, String label) {
                 .children(changeContextMembership.build());
 
         return builder
-                .name("New " + type.getName())
+                .name(DescriptionNameGenerator.getCreationToolName(eClass))
```

Comment:
It seems there is a problem here, you've changed `type.getName` by `eClass`

Reply:
my bad, I fixed it. I didn't use `type` for the icon though, because we don't have all the icons for this right now